src/Codeception/Lib/Generator/GherkinSnippets.php

```diff
@@ -16,22 +16,27 @@ public function {{methodName}}({{params}})
      {
         throw new \Codeception\Exception\Incomplete("Step `{{text}}` is not defined");
      }
-     
+
 EOF;
 
     protected $snippets = [];
     protected $processed = [];
 
-    public function __construct($settings)
+    public function __construct($settings, $test = null)
     {
         $loader = new Gherkin($settings);
+        if (is_null($test)) {
```

Comment:
Complete set https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/
@sergeyklay Better to revert my commit?